templates/education-internships.html

```diff
@@ -15,18 +15,63 @@ <h3>Internships</h3>
+        <a href="https://home.mediquest.nl/" target="_blank>">MediQuest</a>, <a href="https://www.forensicinstitute.nl/" target="_blank">Netherlands Forensic Institute (NFI) Ministry of Justice and Security</a>,
+        <a href="https://nlterm.org/" target="_blank">NL-Term</a>, <a href="https://olery.com/" target="_blank">Olery</a>, <a href="https://www.oracle.com/" target="_blank">Oracle</a>
+        , <a href="https://www.synerscope.com/" target="_blank">Synerscope</a>, <a href="https://www.trivago.com/" target="_blank">Trivago</a>,
+        <a href="https://www.houseofrepresentatives.nl/" target="_blank">House of Representatives (Tweede Kamer der Staten-Generaal)</a>.
     </p>
 
-    <h5>...</h5>
+    <h5>Internship Procedure</h5>
 
     <p>
-        ...
+        Students can choose between a research oriented thesis and a thesis based on an internship in the industry.
+        For both options CLTL has plenty of possibilities.
+    </p>
+    <p>
+        Please find more information on <a href="{{ url_for('education_theses') }}">internships and research topics for Master theses (BA, BSc, MA and MSc) here</a>.
+    </p>
+    <p>
+        <a href="https://vu.nl/en/student/elective-space/internship-or-graduation-project" target="_blank">VUnet login students: information on Internship/thesis</a>
     </p>
 
+    <h5>Documents: Agreements and protocols</h5>
+
+    <ul>
+        <li>Internship Placement Contract / <br> NL: Model stageovereenkomst I / Model stageovereenkomst II</li>
+        <li>Internship Placement Manual / NL: Stagevoorwaarden</li>
+        <li>English version(?)… / NL: Regels verzekering bij stage student</li>
+        <li>Placement Mentor Assessment Form / NL: Beoordelingsformulier stagementor</li>
+    </ul>
+
+    <h5>Practical Information</h5>
+
+    <ul>
+        <li>A member of the programme’s teaching staff wil act as placement supervisor. The student will also be allocated
+            a supervisor at the organization providing the placement: this is the placement mentor.</li>
+        <li>The Examination Board must approve and sign the placement contract.</li>
+        <li>No later than two weeks after the start of the placement, the student will submit a work plan to their supervisor.
+            A work plan details the practical implementation of the placement assignment, including an action plan, a timeline
+            specifying various stages and clearly defined end products. One of the compulsory end products is a placement report.
+            This work plan is produced by the student in consultation with the placement mentor.</li>
+    </ul>
+
+    <h5>How To Apply?</h5>
+
     <p>
-        ...
+        If you want to apply for an internship and/or writing a thesis, you should send <a href="mailto: isa.maks@vu.nl">Dr. Isa Maks</a> an email with your personal
```

Comment:
It would be better to remove the mail address and replace the 'mailto' by a link to the people's page (where the mail addresses are masked to prevent abuse from scrawling)